dependencies.py

```diff
@@ -0,0 +1,44 @@
+import logging
+import sys
+import subprocess
+import json
+
+# Set default logging level to INFO
+logging.basicConfig(level="INFO")
+
+# Check if current python version is 3.8
+if sys.version_info < (3, 8):
+    logging.fatal('McPy needs Python version 3.8.0 or higher to run! Current version is %s.%s.%s' % (
+        sys.version_info.major, sys.version_info.minor, sys.version_info.micro))
+    sys.exit(-4)
+
+# Make sure pip is installed
+logging.info("Making sure pip is installed...")
+subprocess.check_call([sys.executable, '-m', 'ensurepip'])
+logging.info("Pip is installed")
```

Comment:
What happens if it isn't

Reply:
The ensure pip module installs pip if its not already installed.

Reply:
ensurepip installs pip if not installed